### DISCUSSION TITLE
Multi-species analyses

### DISCUSSION THREAD

**Discussion by st-tky** [theislab/scvelo#936] [Q&A]
Hi scvelo team,

Thank you for your great work!
I have been working on scRNA-seq analyses using multispecies datasets containing human and mouse cells (kind of chimera).
I would like to know if it is possible to perform RNA velocity analyses on those multispecies datasets with scVelo (after making a loom file by velocyto package).

Thank you for your consideration.

**Comment by WeilerP** [comment]
@st-tky, in theory, you can run RNA velocity. However, it is currently not clear how to handle batch effects which would be a problem in your case, I suppose.

**Reply by st-tky** [reply]
Thank you for the quick response.
My data contain human and mouse cells in the same batch.
So, I assume that I don't have to handle batch effects in my case.
Is it correct?

**Reply by WeilerP** [reply]
The batch in your data is the species, _i.e._, human and mouse, right?
